frontend/src/pages/home/AlternatingBackgroundColorSectionList.tsx

```diff
@@ -2,45 +2,105 @@ import React from 'react';
 
 import { Grid, useTheme } from '@mui/material';
 
+const SectionGridItem = ({
+  child,
+  color,
+  background,
+  verticalPadding,
+}: {
+  child: React.ReactNode;
+  color: string;
+  background: string;
+  verticalPadding: string;
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Grid
+      item
+      xs={12}
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',
+        padding: verticalPadding,
+        [theme.breakpoints.down('md')]: {
+          padding: `${verticalPadding} 16px ${verticalPadding} 16px`,
+        },
+        color: color,
+        background: background,
+      }}
+    >
+      {child}
+    </Grid>
+  );
+};
+
 const AlternatingBackgroundColorSectionList = ({
   children,
   secondaryBackground = '#1282B2',
   secondaryColor = 'white',
+  verticalPadding = '48px',
+  firstElemVerticalPadding = '32px',
+  lastItemIsPrimary,
 }: {
   children: React.ReactNode[];
   secondaryBackground?: string;
   secondaryColor?: string;
+  // Allow the first item to have a different padding than the rest of the
+  // items.
+  firstElemVerticalPadding?: string;
+  // Used to wrap every item.
+  verticalPadding?: string;
+  // If true the last item will use the primary color regardless of its
+  // position in the list.
+  lastItemIsPrimary?: boolean;
 }) => {
   const theme = useTheme();
 
+  const isLastItem = (idx: number) => {
+    return idx === children.length - 1;
+  };
+
   return (
-    <div
-      style={{
-        width: '100%',
-      }}
-    >
-      <Grid container>
-        {children.map((child: React.ReactNode, i: number) => (
-          <Grid
+    <Grid container width="100%">
+      {children.map((child: React.ReactNode, i: number) =>
+        isLastItem(i) ? (
+          <SectionGridItem
+            key={i}
+            child={child}
+            verticalPadding={verticalPadding}
+            color={
+              lastItemIsPrimary
+                ? theme.palette.text.primary
+                : i % 2 == 0
+                ? secondaryColor
+                : theme.palette.text.primary
+            }
+            background={
+              lastItemIsPrimary
+                ? theme.palette.background.primary
+                : i % 2 == 0
+                ? secondaryBackground
+                : theme.palette.background.primary
+            }
+          />
+        ) : (
+          <SectionGridItem
```

Comment:
This condition is very long and I think not really readable.

You should separate more the element with something like
```
{
   children.slice(0, 1).map(childre, => <>...</>)
}
{
   <>Display last item using "children.at(-1)"</>
}
```

Reply:
Thank you, fixed by https://github.com/tournesol-app/tournesol/pull/1183/commits/c238f4591d3121ff50dc14cc506d536d5b14d196